linera-chain/src/chain.rs

```diff
@@ -445,11 +443,30 @@ where
     }
 
     /// Invariant for the states of active chains.
-    pub fn ensure_is_active(&self) -> Result<(), ChainError> {
+    pub async fn ensure_is_active(&mut self, local_time: Timestamp) -> Result<(), ChainError> {
         if self.is_active() {
             Ok(())
```

Comment:
Maybe less indenting in the rest of the function.

```suggestion
            return Ok(());
```